staging/src/k8s.io/apiserver/pkg/registry/generic/registry/store.go

```diff
@@ -1292,7 +1293,7 @@ func (e *Store) Export(ctx context.Context, name string, opts metav1.ExportOptio
 	if accessor, err := meta.Accessor(obj); err == nil {
 		exportObjectMeta(accessor, opts.Exact)
 	} else {
-		klog.V(4).Infof("Object of type %v does not have ObjectMeta: %v", reflect.TypeOf(obj), err)
+		klog.V(4).InfoS("Have no ObjectMeta", "type", reflect.TypeOf(obj), "err", err)
```

Comment:
According to the migration [instruction](https://github.com/kubernetes/community/blob/master/contributors/devel/sig-instrumentation/migration-to-structured-logging.md), "omit the subject if the subject would be the program itself ". 

In this case, the subject of this log message is the "Object", we shouldn't omit it. 

Maybe this:
```golang
klog.V(4).InfoS("Object of this type does not have ObjectMeta", "type", reflect.Typeof(obj), "err", err)
```

Reply:
This function  has been deleted by https://github.com/kubernetes/kubernetes/pull/98312/files.